Swiftfin/Components/LetterPickerBar/LetterPickerBar.swift

```diff
@@ -0,0 +1,74 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2024 Jellyfin & Jellyfin Contributors
+//
+
+import Defaults
+import SwiftUI
+
+struct LetterPickerBar: View {
+
+    @ObservedObject
+    var viewModel: FilterViewModel
+    private var onSelect: (FilterCoordinator.Parameters) -> Void
+
+    enum Orientation: String, CaseIterable, Defaults.Serializable, Displayable {
+
+        case none
+        case leading
+        case trailing
+
+        var displayTitle: String {
+            switch self {
+            case .none:
+                return L10n.none
+            case .leading:
+                return L10n.left
+            case .trailing:
+                return L10n.right
+            }
+        }
+    }
+
+    @ViewBuilder
+    private var letterPickerBody: some View {
+        VStack(spacing: 0) {
+            ForEach(ItemLetter.allCases, id: \.self) { filterLetter in
+                LetterPickerButton(
+                    filterLetter: filterLetter.value,
+                    activated: viewModel.currentFilters.letter.contains { $0.value == filterLetter.value },
+                    viewModel: viewModel
+                )
```

Comment:
As said in another comment, instead use the environment `isSelected`:

`.environment(\.isSelected, viewModel.currentFilters.letter.contains(filterLetter)`